raster-foundry/raster-foundry
# Update admin UI permission and failure messages

## Overview

This PR updates admin UI permissions and failure messages on platform, organization, and team pages.

~Marked as WIP since platform related pages have not been touched yet.~

**Updates on Jun. 4th:**
 * Finish platform related pages and remove WIP tag

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes

~Platform related pages have not been updated yet.~


## Testing Instructions

 * Log in as a dev user.
 * In a sql shell, update dev user's roles in the public organization, Raster Foundry platform, teams, etc. in the `user_group_roles` table.
 * Navigate through all admin pages as you switch the dev user's role `ADMIN/MEMBER` on platform/org/team levels, check if the authorized/not authorized behaviors are correctly reflected on the frontend and if useful info is shown upon authentication failures.

Closes #3431
